server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/tasks/TaskFactory.java

```diff
@@ -42,7 +42,7 @@
 import spark.Route;
 
 public class TaskFactory {
-    private static final String DEFAULT_PARAMETER = "task";
+    private static final String DEFAULT_PARAMETER = "action";
```

Comment:
do you need to update the documentation for this change? And update the upgrade-notes?

Reply:
In a later commit (sorry) I let the endpoint choose which parameter it is relying on to avoid breacking changes...